native/c/test/makefile

```diff
@@ -184,6 +184,7 @@ build-out/zusf.o \
 build-out/zshmem.o \
 build-out/zowex.test.o \
 build-out/zlogger.test.o \
+build-out/parser.test.o \
```

Comment:
(nitpick) Should we keep the `zlogger` files next to each other? This comment also applies on line 292 below.